jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,71 @@
+const books = require('./books.json')
+
+
+describe('InvertedIndex Class', () => {
+  beforeEach(() => {
```

Comment:
Thanks. I think `beforeAll` will be better, since I will not be calling the methods before each and every spec. With `beforeAll` I will initialize the class and create the indexes once and for all, without repeating myself before each spec.